graphql-docs.gemspec

```diff
@@ -43,7 +43,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'extended-markdown-filter', '~> 0.4'
   spec.add_dependency 'gemoji', '~> 3.0'
   spec.add_dependency 'html-pipeline', '>= 2.14.3', '~> 2.14'
-  spec.add_dependency 'dartsass', '~> 1.49'
+  spec.add_dependency 'sass-embedded', '~> 1.58'
```

Comment:
Latest available as time of writing is `1.71.1`, but setting minimal requirement as `1.58.0`, because it is the last version that supports ruby 2.6, which is the required_ruby_version on gem currently.

Reply:
Sounds great, thank you!